README.md

```diff
@@ -106,25 +103,29 @@ In the Julia REPL, first load the package P4est.jl
 ```julia
```

Comment:
This is the wrong place to attach it, but should we also mention in the README that you need Julia 1.6 to generate the bindings, but Julia 1.7 is fine for using them, and that this is a temporary solution (e.g., "currently it is only possible to generate...")?

Reply:
It's already mentioned in the README:  
> *Note: Currently, `P4est.jl` can only be used with pre-generated bindings on
Julia v1.7. See [issue #39](https://github.com/trixi-framework/P4est.jl/issues/39)
for further discussions.*

Reply:
Yes, but don't we discuss the steps necessary for re-generating the bindings in the README? As a user, I would appreciate it if I would read here "can only be done with Julia 1.6" somewhere. What do you think? If you don't deem it necessary, just resolve this convo.

Reply:
Ah, I see. Added that info in f9c1fb9